timewarp_gl/plugin.cpp

```diff
@@ -228,62 +226,40 @@ class timewarp_gl : public threadloop {
 			}
 		}
 		// Construct a basic perspective projection
-		ksAlgebra::ksMatrix4x4f_CreateProjectionFov( &basicProjection, 40.0f, 40.0f, 40.0f, 40.0f, 0.1f, 0.0f );
-
+		math_util::projection_fov( &basicProjection, 40.0f, 40.0f, 40.0f, 40.0f, 0.1f, 0.0f );
 		// This was just temporary.
 		free(tw_mesh_base_ptr);
 
 		return;
 	}
 
 	/* Calculate timewarm transform from projection matrix, view matrix, etc */
-	void CalculateTimeWarpTransform( ksAlgebra::ksMatrix4x4f * transform, const ksAlgebra::ksMatrix4x4f * renderProjectionMatrix,
-                                        const ksAlgebra::ksMatrix4x4f * renderViewMatrix, const ksAlgebra::ksMatrix4x4f * newViewMatrix )
+	void CalculateTimeWarpTransform( Eigen::Matrix4f * transform, const Eigen::Matrix4f * renderProjectionMatrix,
```

Comment:
Better to obtain references (e.g., `&transform`) to the parameters and not worry about asterisks later on.

Reply:
Sounds good.